BritneyS/SparkleSpin
# Home screen

## What you did :question:
This PR merges all the current features for the Home Screen, which includes the custom button.

## How to test it :microscope:
1. Check out this branch
2. Run the app
3. Note that the home screen displays the title logo for 'Sparkle Spin' and a 'Start' button (see screenshot)

## Any background context you want to provide? :question:
N/A


## Screenshots (if applicable) :camera:
![Simulator Screen Shot - iPhone XR - 2019-03-16 at 18 46 37](https://user-images.githubusercontent.com/8409475/54482642-3a3f7700-481d-11e9-9477-d4bbc39dcc76.png)
